server/middleware/setUpWebSecurity.ts

```diff
@@ -23,14 +23,14 @@ export default function setUpWebSecurity(): Router {
             "'self'",
             (req, res: Response) => `'nonce-${res.locals.cspNonce}'`,
             'code.jquery.com',
-            'www.googletagmanager.com',
-            'www.google-analytics.com',
+            'https://www.googletagmanager.com',
```

Comment:
I don't think this will work (well, CVL don't set these). Not sure why but best to only change what we have to